service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/azure/resourcemanager/storage/StorageManagerOperation.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.workspace.service.resource.controlled.flight.create.azure.resourcemanager.storage;
```

Comment:
Ah ok - these following classes (`StorageManagerOperation`, `CreateStorageAccountRequestData`, etc) will need to move to CRL I think.